src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ArticleSourceModeTests.java

```diff
@@ -280,4 +287,57 @@ public void RTE_019_MoreSympolsTools() {
     source.checkSymbolsTools();
     source.submitArticle();
   }
+
+  @Test(groups = {"RTE_draft","draft_saving"})
+  @Execute(asUser = User.SUS_REGULAR_USER3, onWikia = "draftsavetest")
+  public void RTE_draft_intervening_accepted() {
+    String articleName = PageContent.ARTICLE_NAME_PREFIX + DateTime.now().getMillis();
+//    String articleName2 = articleName+"D";
+
+    SourceEditModePageObject source = new SourceEditModePageObject().openArticle(articleName);
+    source.addContentInSourceMode(articleName);
+//    source.focusTextArea();
+//    source.addContentInSourceMode(articleName);
+//This sleep is necessery for draft to be saved (every 5 sec)
+    new ArticleContent(User.SUS_REGULAR_USER).push("Test content", articleName);
+
+    try {
+      Thread.sleep(5001);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+    }
+    driver.navigate().refresh();
+    AlertHandler.acceptPopupWindow(driver,15);
+    WebElement el = driver.findElement(By.cssSelector(".close.wikia-chiclet-button"));
+    el.click();
+
+    Assertion.assertStringContains(source.getSourceContent(),articleName+articleName);
+    source.submitArticle();
+    driver.quit();
+  }
+
+  @Test(groups = {"RTE_draft","draft_saving"})
+  @Execute(asUser = User.SUS_REGULAR_USER3, onWikia = "draftsavetest")
+  public void RTE_draft_intervening_rejected() {
+    String articleName = PageContent.ARTICLE_NAME_PREFIX + DateTime.now().getMillis();
+    SourceEditModePageObject source = new SourceEditModePageObject().openArticle(articleName);
+    source.addContentInSourceMode(articleName);
+    new ArticleContent().push("Test content", articleName);
+    source.focusTextArea();
+    source.addContentInSourceMode(articleName);
+//This sleep is necessery for draft to be saved (every 5 sec)
+    try {
+      Thread.sleep(5001);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+    }
+    driver.navigate().refresh();
+    AlertHandler.acceptPopupWindow(driver,15);
+    WebElement el = driver.findElement(By.cssSelector(".close.wikia-chiclet-button"));
+    el.click();
+
+//    Assertion.assertStringContains();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)